<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlotMachine</class>
 <widget class="QMainWindow" name="SlotMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>490</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpinLeft">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources/Images.qrc">:/Slots/clover.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="SpinMiddle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources/Images.qrc">:/Slots/diamond.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="SpinRight">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources/Images.qrc">:/Slots/hearts.webp</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources/Images.qrc"/>
 </resources>
 <connections/>
</ui>
